python/dbsp/pipeline.py

```diff
@@ -285,18 +353,27 @@ def status(self) -> Pipeline:
             httpx.TimeoutException: If the request takes longer than Client.timeout.
             dbsp.DBSPServerError: If the DBSP server returns an error.
         """
-        status = get_pipeline.sync_detailed(client=self.api_client, pipeline_id=self.pipeline_id).unwrap(
-            "Failed to retrieve pipeline status")
+        status = get_pipeline.sync_detailed(
+            client=self.api_client, pipeline_id=self.pipeline_id
+        ).unwrap("Failed to retrieve pipeline status")
         return status
 
-    def wait_for_status(self, expected_status: PipelineStatus, timeout: float = sys.maxsize):
-        start = time.time();
+    def wait_for_status(
```

Comment:
are you doing this reformatting by hand or is there some tool?

Reply:
I think this is vscode's doing.